cpp/tests/join/conditional_join_tests.cu

```diff
@@ -384,6 +404,20 @@ TYPED_TEST(ConditionalInnerJoinTest, TestComplexConditionMultipleColumns)
              {{4, 0}, {5, 0}, {6, 0}, {7, 0}});
 };
 
+TYPED_TEST(ConditionalInnerJoinTest, TestSymmetry)
+{
+  auto col_ref_0  = cudf::ast::column_reference(0);
+  auto col_ref_1  = cudf::ast::column_reference(0, cudf::ast::table_reference::RIGHT);
+  auto expression = cudf::ast::expression(cudf::ast::ast_operator::GREATER, col_ref_1, col_ref_0);
+  auto expression_reverse =
+    cudf::ast::expression(cudf::ast::ast_operator::LESS, col_ref_0, col_ref_1);
```

Comment:
Should the `expression_reverse` not use `cudf::ast::ast_operator::LESS_EQUAL`?

Reply:
You're right, that would be more precise but doesn't matter for this particular test input.

Reply:
Just so you know, I am planning to make this change, just waiting to see if a second reviewer adds any further comments so that I can batch them.

Reply:
We were wrong 😂 `LESS_EQUAL` would give you the complement, but we want to strictly get the same set. `LESS` is correct.